docker-compose.yml

```diff
@@ -14,7 +14,9 @@ services:
       replicas: 2
 
   keepalived-exporter:
-    build: .
+    build:
+      context: ./deployments/dev/
```

Comment:
Are you sure it's gonna work with this context?

Reply:
Removed it anyway.